storage/object/api-cli/object-storage-aws-cli.mdx

```diff
@@ -47,8 +47,11 @@ To interact with AWS, `aws-cli` and `awscli-plugin-endpoint` need to be installe
   ```
   aws configure set plugins.endpoint awscli_plugin_endpoint
   ```
-
-3. Open the file in a text editor and edit it as follows:
+3. Create the `~/.aws/config` by running the following command:
+  ```
+  aws configure
+  ```
+4. Open `~/.aws/config` in a text editor and edit it as follows:
```

Comment:
This seems to be repeated twice (same text, though different command, for both steps 2 & 3)